analyzers/its/regression-test.ps1

```diff
@@ -266,11 +266,6 @@ function LoadExpectedIssues($file, $regex){
         throw "Please specify the rule id in the following file: $($file.FullName)"
     }
 
-    # if the ruleId parameter is provided, it should be used to filter the expected issues
```

Comment:
We need all issues to do a proper reporting. Filter is in `VerifyUnexpectedIssues` to be able to implement the logic described on line 321

> There might be the case when different rules fire for the same class. Since we want reduce the noise and narrow the focus,
 we can have only one rule verified per class (this is done by checking the specified id in the first Noncompliant message).